action.yml

```
@@ -44,5 +44,5 @@ outputs:
   changes:
     description: Total number of changes
 runs:
-  using: node12
-  main: action/index.js
+  using: 'node16'
+  main: 'action/index.js'
```

@booklearner this should fix the test failures
```suggestion
  using: node16
  main: action/index.js
```

---

Thanks!